ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js

```diff
@@ -134,20 +134,35 @@ const BaseLine = props => {
             typeof setBufferState === 'function'
               ? setBufferState(unitKey, value)
               : setState({ [unitKey]: value })
+            if (widthKey === 'lineWidth') {
+              setBufferError(
+                validateGeo('lineWidth', {
+                  value: props[widthKey],
+                  units: value,
+                })
+              )
+            }
           }}
         >
           <TextField
             type="number"
             label="Buffer width"
             value={String(props[widthKey])}
             onChange={value => {
-              if (widthKey === 'lineWidth') {
```

Comment:
❓ Just curious as to why setting the error state has been removed from the onChange here but added into the onChange in point-radius?https://github.com/codice/ddf/pull/5814/commits/8ddacff3a14d526e65ce956598108acf9acb2902#diff-d97d16e1ef7ed34471eedc4acec209a7R80

Reply:
I made this change because of the different way we're checking if there is an error or not.. before, we were just doing it if the buffer was 0 or empty. now we're doing it based on that conversion (if its less than one meter or equivalent). because it's more complicated now I moved this validation to the onBlur.